.ruff.toml

```diff
@@ -0,0 +1,39 @@
+include = ["ensysmod/**/*.py", "tests/**/*.py"]
+
+line-length = 150
+target-version = "py311"
+
+[lint]
+select = ["ALL"]
+ignore = ["N", "D", "ANN", "S", "BLE", "A", "EM", "TD", "FIX", "PL", "TRY"]
+
+# conflicting lint rules: https://docs.astral.sh/ruff/formatter/#conflicting-lint-rules
+extend-ignore = [
+    "W191",
+    "E111",
+    "E114",
+    "E117",
+    "D206",
+    "D300",
+    "Q000",
+    "Q001",
+    "Q002",
+    "Q003",
+    "COM812",
+    "COM819",
+    "ISC001",
+    "ISC002",
+]
+
+[lint.per-file-ignores]
+"__init__.py" = [
+    "I001", # allow unsorted imports in __init__.py
+    "F401", # allow unused imports in __init__.py
+]
+"tests/*" = [
+    "S101", # allow assert statements for pytest
+    "ARG",  # allow unused arguments for pytest fixtures
+]
+"ensysmod/api/endpoints/*" = [
+    "B008", # allow function calls (Depends, File) in API endpoints
+]
```

Comment:
can't we put this into the pyproject.toml too - instead of having another config? :)

Reply:
Done